components/dropdown/dropdown-content-mixin.js

```diff
@@ -424,7 +456,7 @@ export const DropdownContentMixin = superclass => class extends RtlMixin(supercl
 			});
 
 			const spaceRequired = {
-				height: contentRect.height + 20,
+				height: contentRect.height + headerRect.height + footerRect.height + 10,
```

Comment:
Curious what the extra `10px` is?

Reply:
I'm not entirely sure, I removed 10px to account for the header/footer heights (they were 5px before and still default to it, but since we're measuring we didn't need them) and this worked with the visual diff. @AllanKerr might have a better idea where they come from?

Reply:
My guess: `5px` and `5px` was from the old header and footer. The little triangle thing is `18px` but it's positioned `-10px` so would take up an extra `8px` of height... but inspecting it appears to be closer to `25px` high positioned up `-10px`... so I'm not really sure because that would be `15px` plus the other 10 making it `25px` instead of `20px`.

Reply:
That aligns with my best guess as well. I just pulled through the same value to keep it consistent with the Polymer version. It shouldn't make a huge difference since all its used to determine is whether to place the dropdown above or below the opener.